subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -56,6 +57,9 @@ def setUp(self):
         self.app.prober = mock.Mock()
         self.fsc = FilesystemController(app=self.app)
         self.fsc._configured = True
+        # _start() would do this normally, but the rest of what _start() does
+        # is going to need mocking out.
+        self.fsc.autoinstall_applied = asyncio.Event()
```

Comment:
eh, the assignment is done in `__init__`? is this a leftover from a previous version?

Reply:
This is a python 3.8 vs 3.10 distinction.  I originally had this initialized in `__init__`, but py3.8 is grumpy about the loop object.  So instead I need this worse version that risks a race.

Reply:
ah.

Reply:
What's the problem exactly in Python 3.8? Is it because in Python < 3.10, asyncio.Event instances must be created after the event loop is running? 

https://github.com/canonical/subiquity/pull/1446#issuecomment-1294792866
https://github.com/canonical/subiquity/pull/1446#issuecomment-1293254597

Reply:
> What's the problem exactly in Python 3.8? Is it because in Python < 3.10, asyncio.Event instances must be created after the event loop is running?

Yes.  See a Focal CI run, where I attempted to create the Event in __init__, for some test failures.
https://github.com/canonical/subiquity/actions/runs/4166633969/jobs/7211213000#step:3:2839